src/dscim/menu/main_recipe.py

```diff
@@ -318,6 +318,8 @@ def scc():
             self.logger.info("Processing SCC calculation ...")
             if self.fit_type == "quantreg":
                 self.full_uncertainty_iqr
+                self.calculate_scc
```

Comment:
I wasn't sure exactly how to work with this since it's kind of a weird quirk of the `order_plate` function. In the original `scc` function which is a "course" of the `order_plate` function, it will only calculate `full_uncertainty_iqr` (if quantreg). This meant that if I wanted statistical uncertainty, I couldn't go through the `order_plate` function. There are a few potential solutions I can think of:

- Add a new attribute to dscim that specifies uncertainty range (i.e. stat or full) 
- Replace this scc function with a function that calls recipe functions according to which `save_files` you have specified
- Replace the two uncertainty functions with a single function that feeds the scc (collapsed for stat uncertainty or uncollapsed for full uncertainty) into it based on `fair_aggregation`
- Add a few more "courses" that allow the user to select which type of scc they want

Reply:
Let's try the first option "Add a new attribute to dscim that specifies uncertainty range (i.e. stat or full)" in a later PR